scripts/kallisto2ga.py

```diff
@@ -0,0 +1,135 @@
+"""
+    Script to parse the abundances.txt output file produced by kallisto.  As
+    current version of the rnaQuant is using a flat file for the data this
+    parser is living in server/scripts.  Once the backend store is ready for
+    rnaQuant then this functionality will move to server/loaders.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import sys
+import errno
+
+
+def makeDir(path):
+    try:
+        os.makedirs(path)
+    except OSError as exception:
+        if exception.errno != errno.EEXIST:
+            raise
+
+
+# TODO: placeholder values need to be calculated then removed
+def getCount(expressionId):
+    rawCount = 0
+    return "%d" % rawCount
```

Comment:
We tend to use the formatting mini-language in favour of string interpolation. This would be 
```python
return "{}".format(rawCount)
```
It's a bit more flexible, and consistency is nice.